docs/proposals/version-documentation.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "Documentation Versioning"
+linkTitle: "Documentation Versioning"
+weight: 1
+slug: documentation-versioning
+---
+
+- Author: [Jay Batra](https://github.com/jaybatra26)
+- Date: March 2020
+- Status: proposal
+
+## Problem
+In Cortex, currently, we are missing versioning of documentation. The idea is to have version documentation just like Prometheus.[`Prometheus`](https://prometheus.io/docs/introduction/overview/). Documentation is the main source of information for current contributors and first-timers. A properly versioned documentation will help everyone to have a proper place to look for answers before flagging it in the community. 
+
+In this proposal, we want to solve this. In particular, we want to:
+1. Version Documentation.
+2. Add or remove necessary and unnecessary data from it.
+3. Organize the documentation according to the latest and stable developments.
+
+
+## Proposed solution
+Currently, the documentation is residing under the docs/ folder of cortexproject/cortex. It is built by Hugo using the theme [`docsy`](https://www.docsy.dev). The documentation should be a separate project and should go inside of cortexproject/docs. It will have a proper [`drop-down menu`](https://www.docsy.dev/docs/adding-content/versioning/#adding-a-version-drop-down-menu) which will enable proper versioning. It has a section [`params.version`](https://www.docsy.dev/docs/adding-content/versioning/#adding-a-version-drop-down-menu) in config.toml which will allow us to map URLs with proper versions. We will have to change all the occurrences of older doc links with new links. The major versions would contain the releases, all other work should be a part of minor versions. The major version here referred is 1.0.0 and minor versions should be like 1.x
```

Comment:
```
The documentation should be a separate project and should go inside of cortexproject/docs
```

What's the rationale for this? What advantage do we have if we put it into a separate repo?

Reply:
@weeco Just for the organization. Prometheus has a separate project for its documentation. Also, since this is a complete website, it should have its own place in the org.